### ISSUE TITLE
Wookie handler seems broken

### ISSUE THREAD

**Issue by andy128k** [fukamachi/clack#108]
I have problems with clack+wookie server.
I do: 

```lisp
(ql:quickload :clack)
(clack:clackup
    #'(lambda (env) '(200 (:content-type "text/plain") ("Hello, Clack!")))
    :server :wookie)
```

Browser receives first response only, but when i hit F5 it says 'Unable to connect'.

It seems like `as:socket-closed` signal kills server.

**Comment by Rudolph-Miller** [comment]
Can you retry with this patch https://github.com/fukamachi/clack/pull/109 ?

**Comment by andy128k** [comment]
@Rudolph-Miller your patch fixes my problem.

**Comment by Rudolph-Miller** [comment]
That's good to hear.

**Comment by fukamachi** [comment]
Merged #109.
